packages/documentation-site/patternfly-docs/content/design-guidelines/styles/spacers/spacers.css

```
@@ -57,23 +57,23 @@
 }
 
 .ws-content-correct {
-  color: #486b00;
+  color: var(--pf-v5-global--info-color--100);
```

Did you mean to turn this content blue? It was green before. 

```suggestion
  color: var(--pf-v5-global--success-color--100);
```